syntax = "proto3";

package model;

option java_package = "com.jason.mlsvconnx.proto";
option java_multiple_files = true;

service Model {
  rpc RunModel (ModelRequest) returns (ModelReply) {}
}

message ModelRequest {
  repeated int64 input1 = 1;
  repeated float input2 = 2;
}

message ModelReply {
  repeated float output = 1;
}